src/test/samVisualize/withoutExtension.ts

```diff
@@ -0,0 +1,12 @@
+/*!
```

Comment:
Naming the file after its sole method is (in this case) not very clear and makes it difficult to expand.

Reply:
Yeah, I'll replace it with something like `samVisualizeTestUtil.ts` to clear up intention + allow for additions

Reply:
Will do!